sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -520,3 +563,17 @@ def test_adaboost_consistent_predict(algorithm):
         np.argmax(model.predict_proba(X_test), axis=1),
         model.predict(X_test)
     )
+
+
+@pytest.mark.parametrize(
+    'model, X, y',
+    [(AdaBoostClassifier(), *datasets.load_iris(return_X_y=True)),
```

Comment:
iris and boson are both loaded in this file already, could use those instead of loading them again I think.